src/frontend/src/common/styles/basicStyle.ts

```diff
@@ -82,5 +82,8 @@ export const transparentScrollbars = (): string => {
     &::-webkit-scrollbar-track {
       background: transparent;
     }
+    &::-webkit-scrollbar {
+      display: none;
+    }
```

Comment:
@mmmmmmaya I'm not sure why I needed this - the transparent background wasn't working.  Are there any red flags with adding this?  It definitely still scrolls.